routes/adminapi.js

```diff
@@ -1,34 +1,34 @@
 const express = require('express');
 const https = require('https');
-const MongoClient = require('mongodb').MongoClient;
-
 const router = express.Router();
-// Connection URL
-const url = 'mongodb://localhost:27017/myproject';
-let collections;
-// Use connect method to connect to the Server
-MongoClient.connect(url, (err, db) => {
-	if (err === null) {
-		collections = db.collection('documents');
-	}
+const bodyParser = require('body-parser');
```

Comment:
y is this not enough?
https://github.com/prateekbh/Guess/blob/master/app.js#L22